### PR TITLE
Sort some more badges

### PR DESCRIPTION
**Pull request by paulmelnikow**
- Move the Docker Pulls badge from **Other** to **Downloads**.
- Create **Platform & Language Support** for the following:
    - django version support
    - python version support
    - node version support
    - type support
    - pypi - wheel (waiting to avoid conflict with #1922)
    - pipi - format (waiting to avoid conflict with #1922)
    - pipi - implementation (waiting to avoid conflict with #1922)
    - hhvm
    - cocoapods platform

Feedback is welcomed!

Ref: #1905

### CODE REVIEW

File: services/npm/npm-type-definitions.service.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const NpmBase = require('./npm-base')
 // package.json.
 module.exports = class NpmTypeDefinitions extends NpmBase {
   static get category() {
-    return 'version'
+    return 'platform-support'
   }
```
Comment (paulmelnikow) [inline services/npm/npm-type-definitions.service.js]: The diffs in the new-style services are much more readable. ✔️

**Comment by tooomm** [comment]
>Thanks for contributing to our documentation. We ❤️ our documentarians!

Where did you change documentation in this PR?! I'm confused...

**Comment by paulmelnikow** [comment]
Ah, good call on the PHP versions and conda platform. I’ll update this.

Agreed on PyPI too. I’m waiting on #1922 to avoid conflicts.

**Comment by paulmelnikow** [comment]
Right now it's called **Platform and Language Support** with the goal of trying to pull in the npm type defs badge, which is a horse of its own special color. Maybe **Platform and Version Support** is better, and we place the npm defs badge in **Other**?

**Comment by paulmelnikow** [comment]
Updated!

**Comment by paulmelnikow** [comment]
Yea, that's what I was thinking too: that we'd merge this without touching PyPI, and then adjust the PyPI categorization in #1922. Thanks!

**Pull request merged**